src/transactions.rs

```diff
@@ -383,25 +406,143 @@ macro_rules! create_tx {
     }
 }
 
+fn weight_to_kweight_roundup(weight: u64) -> u64 {
```

Comment:
not really a roundup :sweat_smile: `assert_eq!(weight_to_kweight_roundup(1000), 1);` panicked

Reply:
Woops, i somehow dropped the `-1`.. Fixed, thanks!